.github/foss_project_collector.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/env bash
```

Comment:
Does this _need_ to be a shell script? I can't scratch the itch that this is much more complicated than it needs to be, considering that we have strict rules on how to add a new project to the pool. It might just be down to the fact that I find shell scripts just much harder to decypher against a python script (even if its a few lines lengthier).